### PR TITLE
merge Power changes

### PR DESCRIPTION
**Pull request by drossetti**
This is to close issue #5

### CODE REVIEW

**Comment by drossetti** [comment]
@sjeaugey could you please review ?

File: gdrdrv/gdrdrv.c [hunk 1]
```diff
@@ -43,8 +48,15 @@ static inline pgprot_t pgprot_modify_writecombine(pgprot_t old_prot)
...
+#define get_tsc_khz() cpu_khz // tsc_khz
+#elif defined(CONFIG_PPC64)
+static inline pgprot_t pgprot_modify_writecombine(pgprot_t old_prot)
+{
+    return pgprot_writecombine(old_prot);
+}
+#define get_tsc_khz() (get_cycles()/1000) // dirty hack
 #else
-#error "X86_64/32 is required"
+#error "X86_64/32 | PPC64 is required"
```
Comment (sjeaugey) [inline gdrdrv/gdrdrv.c]: "or PPC64" would be better since it is a comment, not code.

Comment (drossetti) [inline gdrdrv/gdrdrv.c]: right!

**Comment by drossetti** [comment]
resolve issue #5

**Pull request merged**